### PR TITLE
Warrior cleanups

### PR DESCRIPTION
**Pull request by drewmoseley**

### CODE REVIEW

**Comment by texierp** [comment]
Hello @drewmoseley 

FYI, I'm in the process of updating both meta-mender-nxp and
meta-mender-atmel for warrior ;)

Also, it seems now interesting to define a sub-maintainer for each
meta layer, e.g. bump, build failure and so on ...

@mirzak any thoughts ?

In any case, I am ok to maintain nxp and atmel layers.

Pierre-Jean

File: README.md [hunk 1]
```diff
@@ -23,19 +23,6 @@ boards.
...
-- meta-sunxi
-- meta-rockchip
-- meta-renesas
-
-Vendor-oriented:
-
-- meta-variscite
-- meta-phytec
-- meta-toradex-nxp
-
```
Comment (mirzak) [inline README.md]: Definitely want to keep everything around and give people a chance to update. So drop this commit

File: meta-mender-update-modules/recipes-mender/mender-update-modules/mender-update-modules.inc [hunk 2]
```diff
@@ -4,6 +4,6 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=86d3f3a95c324c9479bd8986968f4327"
 
 SRC_URI = "git://github.com/mendersoftware/mender-update-modules.git"
 
-SRCREV = "7631f32b54d06c56f786b1459e5fce158efba14a"
+SRCREV = "033fd92fa1e8d9c19cc3f3e00eadf7800c5dcc00"
```
Comment (mirzak) [inline meta-mender-update-modules/recipes-mender/mender-update-modules/mender-update-modules.inc]: Can you do a cherry-pick of this one?

File: meta-mender-update-modules/recipes-mender/mender-update-modules/mender-ipk_git.bb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+           bb.error("Select IPK style packaging using the following:")
+           bb.error("     PACKAGE_CLASSES = \"package_ipk\"")
+}
+
+do_install_class-target() {
+    install -d ${D}/${datadir}/mender/modules/v3
+    install -m 755 ${S}/ipk/module/ipk ${D}/${datadir}/mender/modules/v3/ipk
+}
+
+FILES_${PN} += "${datadir}/mender/modules/v3/ipk"
```
Comment (mirzak) [inline meta-mender-update-modules/recipes-mender/mender-update-modules/mender-ipk_git.bb]: Can you do a cherry-pick of this one

**Comment by mirzak** [comment]
> Also, it seems now interesting to define a sub-maintainer for each
meta layer, e.g. bump, build failure and so on ...

> @mirzak any thoughts ?

I think it is a good idea and I have for long thought about doing something with the README files, that is consistent across layers.

I will create a draft for review for one of "my" layers, and then we can spread it out.

**Comment by texierp** [comment]
> > Also, it seems now interesting to define a sub-maintainer for each
> > meta layer, e.g. bump, build failure and so on ...
> 
> > @mirzak any thoughts ?
> 
> I think it is a good idea and I have for long thought about doing something with the README files, that is consistent across layers.
> 
> I will create a draft for review for one of "my" layers, and then we can spread it out.

Thanks @mirzak, PR #102 is now updated with the revised README ;)

**Comment by drewmoseley** [comment]
@mirzak updated per your comments.

**Pull request merged**